src/easylink/configuration.py

```diff
@@ -49,15 +49,19 @@ def load_params_from_specification(
     }
 
 
-def _load_input_data_paths(input_data_specification_path: Union[str, Path]) -> List[Path]:
+def _load_input_data_paths(
```

Comment:
this reproduces changes from Milestone 2.8, just using the input data dictionary keys to index the file inputs in the pipeline schema constants.